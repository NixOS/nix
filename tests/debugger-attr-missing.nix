# with import ./config.nix;

let 
  testattrs = { a = 1; b = 2; };
  wat = arg : arg + testattrs.c;
  foo = arg : arg2 : arg + arg2;
  
in

foo 12 (wat 11)


# mkDerivation {
#   name = builtins.break "simple";
#   builder = ./simple.builder.sh;
#   PATH = "";
#   goodPath = path;
# }
