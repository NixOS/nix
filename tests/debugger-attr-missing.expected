[34;1m0[0m: [31;1merror: [0mattribute '[35;1mc[0m' missing
[34;1m1[0m: while evaluating the attribute '[35;1mc[0m'
[34;1m2[0m: while calling [35;1m'wat'[0m
[34;1m3[0m: while calling [35;1m'foo'[0m
[34;1m4[0m: while evaluating the file '[35;1m/home/bburdette/op-code/nix/tests/debugger-attr-missing.nix[0m':
